.changelog/2592.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:enhancement
+handling "404 Not Found" errors during the deletion of Kubernetes resources, particularly in cases where the resource may have already been deleted by an operator managing the CRD before Terraform attempts to delete it.
```

Comment:
I would add a reference to the manifest resource here, to make it clear this change only affects its behaviour.
```suggestion
`kubernetes_manifest` - handling "404 Not Found" errors during the deletion of Kubernetes resources, particularly in cases where the resource may have already been deleted by an operator managing the CRD before Terraform attempts to delete it.
```